README.md

```diff
@@ -119,6 +123,72 @@ are installed:
 The `opam` binary is added to the `PATH` for subsequent actions, so that
 executing `opam` commands will just work after that.
 
+## Advanced Configurations
+
+It is possible to feed different values to `opam-repository` depending on the
+platform the runner. The syntax of Github's workflows is flexible enough to
+offer several methods to do this.
+
+For example, using the strategy matrix:
+
+```
+runs-on: ${{ matrix.os }}
+strategy:
+  matrix:
+    os: [macos-latest, windows-latest, ubuntu-18.04]
```

Comment:
I think it's better to use `ubuntu-latest` in the documentation to keep it up to date.